src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
```

Comment:
I found some formatting tags. Should they be removed?
![image](https://user-images.githubusercontent.com/60385689/97513440-3f881000-19c7-11eb-97c9-160f2c237929.png)
![image](https://user-images.githubusercontent.com/60385689/97513452-4a42a500-19c7-11eb-9f55-985601c5dab5.png)